Main/Source/char.cpp

```diff
@@ -11972,6 +11972,22 @@ void character::ReceiveMustardGasLiquid(int BodyPartIndex, long Modifier)
                             BodyPartIndex, YOURSELF, false, false, false);
       CheckDeath(CONST_S("killed by a fatal exposure to mustard gas"));
     }
+
+    if(IsPlayer())
+    {
+      action* Action = GetAction();
+
+      if(Action && Action->IsRest() && !Action->InDNDMode()
+         && BodyPartIsVital(BodyPartIndex) && BodyPart->IsBadlyHurt())
+      {
+        ADD_MESSAGE("You're about to die from mustard gas.");
+
+        if(game::TruthQuestion(CONST_S("Try to find a healer? [Y/n]"), YES))
```

Comment:
"Try to find a healer?" may be in dungeons we will not have the time to reach one, what about "I should try to heal myself promptly! confirm? [y/n]" or something in that line. Or it could scan for a healer and open the minimap highlighting him/her xD, interesting that I remember a valpurus (?) angel putting back my arms quite often :>

Reply:
It seems no NPC has the ability to heal you... potions do not seem very helpful... resting in place is too slow. At least you should keep moving and eat something like magical mushrooms. So "Go see if there is a healing potion nearby? [Y/n]" ?

Reply:
or "Stop to heal before it is too late? [Y/n]", I think many of these details (that are not the core code logic) end being a matter of preference and are not that important, in a sense that it is not a requirement to merging at all and are extremely easy to be changed later. :)

Reply:
I feel "Try to find a healer?" is quite good, though actualy they don't provide any cure for mustard gas damage. :p